src/dom/html/WindowRe.re

```diff
@@ -44,17 +44,17 @@ module Impl = (T: {type t;}) => {
   [@bs.get] [@bs.return nullable] external opener : t_window => option(Dom.window) = "";
   [@bs.get] external outerWidth : t_window => int = "";
   [@bs.get] external outerHeight : t_window => int = "";
-  [@bs.get] external pageXOffset : t_window => int = ""; /* alias for screenX */
-  [@bs.get] external pageYOffset : t_window => int = ""; /* alias for screenY */
+  [@bs.get] external pageXOffset : t_window => float = ""; /* alias for scrollX */
+  [@bs.get] external pageYOffset : t_window => float = ""; /* alias for scrollY */
```

Comment:
These are actually aliases of `scrollX` and `scrollY`, so I corrected the comments.

Reply:
Oops. Thanks :)